### DISCUSSION TITLE
Contribution gamification

### DISCUSSION THREAD

**Discussion by ameershah48** [farhan-helmy/ratemysurau#14] [Ideas]
- Record user's place contributions/submission 
- Build a rank page
- List down most contributed users in the rank page 
- Reward badge to users for reaching certain milestones 

This can encourage others to contribute as well

**Comment by farhan-helmy** [comment]
menarik ni ameer, jom buat

**Reply by ameershah48** [reply]
lepas dh implement user auth kita tgk mcm mana haha

**Reply by farhan-helmy** [reply]
bolehhh, aku plan pakai google auth dulu

**Reply by farhan-helmy** [reply]
aku rasa senang je ni, just update schema relatekan user dengan upload surau, letak logic if user logged in boleh relatekan

**Reply by ameershah48** [reply]
iya, sbb mcm tak kacau sgt schema sedia ada

**Reply by farhan-helmy** [reply]
kena fikir user boleh buat apa dengan contribution point tu

**Comment by ismi-abbas** [comment]
#5 Macam sama dengan idea Atif
